cdk/domino_cdk/eks_stack.py

```diff
@@ -498,7 +550,6 @@ def provision_unmanaged_nodegroup(
 
         scope = cdk.Construct(self, f"UnmanagedNodeGroup{name}")
         for i, az in enumerate(self.vpc.availability_zones[:max_nodegroup_azs]):
-
             indexed_name = f"{self.name}-{name}-{i}"
```

Comment:
This the area I mentioned in my other comment where I think we could switch to az. The `f"ASG{i}"` below also seems kind of bad, as we're naming multiple ASGs something non-unique. Apparently doesn't cause problems because it's been working, but still... throwing the asg name in as well (the model used for managed) would be better for that I think.

Reply:
ok

Reply:
I'll try to change ASG{i} to indexed_name and see how it lives

Reply:
Works fine. ID is not actually visible anywhere .

See https://us-west-2.console.aws.amazon.com/ec2autoscaling/home?region=us-west-2#/details/denis-test1-compute-us-west-2a?view=details

Logical ID UnmanagedNodeGroupcomputedenistest1compute0ASG5CC272DB appears in tags so id field is not entorely lost